### PR TITLE
Adding error from machine startup and installers to slf4j log

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Adding  error from machine startup and installers to  slf4j log.

Added default logback configuration to reduce log output during tests.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/9641

#### Release Notes
Error from machine startup and installers are put to che-server logs.

#### Docs PR
n/a

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

File: wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorInstallerLogEventLogger.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * The goal of this class it to catch all InstallerLogEvent events from error stream and dump it to
+ * slf4j log.
+ */
+@Singleton
+public class ErrorInstallerLogEventLogger implements EventSubscriber<InstallerLogEvent> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ErrorInstallerLogEventLogger.class);
+
+  @Inject private EventService eventService;
```
Comment (sleshchenko) [inline wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorInstallerLogEventLogger.java]: I guess you don't need this field. You can use @Inject for method instead:
```
  @Inject
  public void subscribe(EventService eventService) {
    eventService.subscribe(this, InstallerLogEvent.class);
  }
```

File: wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorInstallerLogEventLogger.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  @PostConstruct
+  public void subscribe() {
+    eventService.subscribe(this, InstallerLogEvent.class);
+  }
+
+  @Override
+  public void onEvent(InstallerLogEvent event) {
+    InstallerLogEvent.Stream stream = event.getStream();
+    String text = event.getText();
+    if (stream != null && stream == InstallerLogEvent.Stream.STDERR && !isNullOrEmpty(text)) {
```
Comment (sleshchenko) [inline wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorInstallerLogEventLogger.java]: I would say that `stream != null` is redundant part

File: wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorMachineLogEventLogger.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  @PostConstruct
+  public void subscribe() {
+    eventService.subscribe(this, MachineLogEvent.class);
+  }
+
+  @Override
+  public void onEvent(MachineLogEvent event) {
+    String stream = event.getStream();
+    String text = event.getText();
+    if (!isNullOrEmpty(stream) && "stderr".equalsIgnoreCase(stream) && !isNullOrEmpty(text)) {
```
Comment (sleshchenko) [inline wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorMachineLogEventLogger.java]: I think `!isNullOrEmpty(stream)` can be removed

File: wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorMachineLogEventLogger.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      RuntimeIdentityDto identity = event.getRuntimeId();
+      LOG.error(
+          "Machine {} error from owner={} env={} workspace={} text={} time={} ",
```
Comment (sleshchenko) [inline wsmaster/che-core-api-logger/src/main/java/org/eclipse/che/api/logger/ErrorMachineLogEventLogger.java]: Maybe it makes sense to add '' for parameters. I mean > "Machine \`{}\` error from owner=\`{}\` ..."

**Review by ghost — APPROVED** [review]

**Review by sleshchenko — APPROVED** [review]

**Review by riuvshin — APPROVED** [review]
<3

**Comment by skabashnyuk** [comment]
ci-test

**Pull request merged**